### ISSUE TITLE
Outdated sdk licenses

### ISSUE THREAD

**Issue by Poldan** [beevelop/docker-cordova#53]
On your image got erros about not accepted licenses.
```
FROM beevelop/cordova

RUN 
	echo y | $ANDROID_HOME/tools/bin/sdkmanager "platforms;android-28" && \
	echo y | $ANDROID_HOME/tools/bin/sdkmanager "build-tools;28.0.3"
```
This dockerfile make all works good.

**Comment by arek159** [comment]
problem still exists while building ionic app

**Comment by beevelop** [comment]
Should be fixed with `v2021.02.1` ✅ Keep in mind, that you might need to accept the Android licenses before executing your own scripts.
